index.html

```diff
@@ -6,7 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
-  <title>Daniel</title>
+  <title>Daniel Portfolio</title>
```

Comment:
- [ ] Great Job So far but There is something you still need to work on for your website to be accessible, I noticed that some of the text in your code has very low contrast with the background. This can make it difficult for users, especially those with visual impairments, to read the text. Can you please adjust the contrast to ensure that the text is legible and accessible for all users? :wink:  if you wanna check that use this tool [Download this chrome extention](https://wave.webaim.org/extension/)